packages/stark-ui/assets/styles/_variables.scss

```diff
@@ -0,0 +1,237 @@
+  A700: #c51162,
+  contrast: (
+    50: $dark-primary-text,
+    100: $dark-primary-text,
+    200: $dark-primary-text,
+    300: $dark-primary-text,
+    400: $dark-primary-text,
+    500: $light-primary-text,
+    600: $light-primary-text,
+    700: $light-primary-text,
+    800: $light-primary-text,
+    900: $light-primary-text,
+    A100: $dark-primary-text,
+    A200: $light-primary-text,
+    A400: $light-primary-text,
+    A700: $light-primary-text
+  )
+);
+
+$warning-palette: (
+  50: #ffccbc,
+  100: #ffcdd2,
+  200: #ef9a9a,
+  300: #e57373,
+  400: #ef5350,
+  500: #f44336,
+  600: #e53935,
+  700: #d32f2f,
+  800: #c62828,
+  900: #b71c1c,
+  A100: #ff8a80,
+  A200: #ff5252,
+  A400: #ff1744,
+  A700: #d50000,
+  contrast: (
+    50: $dark-primary-text,
+    100: $dark-primary-text,
+    200: $dark-primary-text,
+    300: $dark-primary-text,
+    400: $dark-primary-text,
+    500: $light-primary-text,
+    600: $light-primary-text,
+    700: $light-primary-text,
+    800: $light-primary-text,
+    900: $light-primary-text,
+    A100: $dark-primary-text,
+    A200: $light-primary-text,
+    A400: $light-primary-text,
+    A700: $light-primary-text
+  )
+);
+
+$backgrounds: (
+  status-bar: map_get($mat-grey, 300),
+  app-bar: map_get($mat-grey, 100),
+  background: #f6f9fa,
+  hover: rgba(black, 0.04),
+  card: #ffffff,
+  dialog: #ffffff,
+  disabled-button: rgba(black, 0.12),
+  raised-button: #ffffff,
+  focused-button: $dark-focused,
+  selected-button: map_get($mat-grey, 300),
+  selected-disabled-button: map_get($mat-grey, 400),
+  disabled-button-toggle: map_get($mat-grey, 200),
+  unselected-chip: map_get($mat-grey, 300),
+  disabled-list-option: map_get($mat-grey, 200)
+);
+
+$foregrounds: (
+  base: black,
+  divider: $dark-dividers,
+  dividers: $dark-dividers,
+  disabled: $dark-disabled-text,
+  disabled-button: rgba(black, 0.26),
+  disabled-text: $dark-disabled-text,
+  hint-text: $dark-disabled-text,
+  secondary-text: $dark-secondary-text,
+  icon: rgba(black, 0.54),
+  icons: rgba(black, 0.54),
+  text: rgba(black, 0.87),
+  slider-min: rgba(black, 0.87),
+  slider-off: rgba(black, 0.26),
+  slider-off-active: rgba(black, 0.38)
+);
+
+$base-theme: (
+  primary-palette: mat-palette($primary-palette),
+  accent-palette: mat-palette($accent-palette),
+  warn-palette: mat-palette($mat-red)
+);
+
+$button-theme: (
+  border-radius: $border-radius,
+  font-size: 14px,
+  font-weight: 500,
+  line-height: 36px,
+  padding: 0 16px,
+  small-font-size: 12px,
+  small-font-weight: 500,
+  small-line-height: 26px,
+  small-min-width: 50px,
+  small-padding: 0 5px,
+  large-font-size: 30px,
+  large-font-weight: 500,
+  large-line-height: 52px,
+  large-min-width: 88px,
+  large-padding: 0 20px
+);
+
+$card-theme: (
+  border-radius: $border-radius
+);
+
+$table-theme: (
+  border-radius: $border-radius
+);
+
+//font-size, line-height, font-weight
+$typography-theme: (
+  font-family: 'Roboto, "Helvetica Neue", sans-serif',
```

Comment:
The fonts should be replaced by the ones we have in the old stark in `src/ui/css/_base-styles.pcss`:
```css
/********** TYPOGRAPHY **********/
body {
  /* system sans-serif font stack */
  font-family: -apple-system, BlinkMacSystemFont,
  Segoe UI, Roboto, Oxygen,
  Ubuntu, Cantarell, Fira Sans,
  Droid Sans, Helvetica Neue, sans-serif;
  font-size: 13px;
}
```